python/nano/src/bigdl/nano/utils/inference/pytorch/model.py

```diff
@@ -62,5 +59,23 @@ def numpy_to_tensors(np_arrays):
             tensors = tensors[0]
         return tensors
 
+    def dump_status(self, path):
+        with open(path + "/meta-data.yml", 'w') as f:
+            yaml.safe_dump(self.status, f)
+
     def save(self, path):
```

Comment:
We should not have `save` and `load` methods in models; if needed, change to `_save` and `_load`

Reply:
Do you mean users are not allowed to save model directly by `model.save`? They must use `trainer.save`.

Reply:
renamed save/load

Reply:
> Do you mean users are not allowed to save model directly by `model.save`? They must use `trainer.save`.

Yes - it will be confusing if you provide multiple APIs for doing the same thing